xgi/drawing/draw_utils.py

```diff
@@ -103,8 +103,8 @@ def _parse_color_arg(colors, ids, id_kind="edges"):
     --------
     colors : single color or ndarray
         Processed color values for plotting.
-    colors_are_mapped : bool
-        True if the colors are mapped and need special handling. This
+    colors_to_map : bool
```

Comment:
can this just be called `map_colors`?

Reply:
Yea that would work too I guess, but the colors are only mapped later in the draw function